src/test/java/com/buyanywhere/productcatalog/dto/CategoryDtoTest.java

```diff
@@ -0,0 +1,100 @@
+package com.buyanywhere.productcatalog.dto;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+public class CategoryDtoTest {
+
+    private CategoryDto categoryDto;
+
+    @Before
+    public void setup(){
+        categoryDto = new CategoryDto();
```

Comment:
There is no point of having this initialized here not having the field categoryDTO.